pkgs/tools/filesystems/hubicfuse/default.nix

```diff
@@ -1,12 +1,14 @@
-{ stdenv, fetchurl, pkgconfig, curl, openssl, fuse, libxml2, json_c, file }:
+{ stdenv, fetchFromGitHub, pkgconfig, curl, openssl, fuse, libxml2, json_c, file }:
 
 stdenv.mkDerivation rec {
   name = "hubicfuse-${version}";
-  version = "2.1.0";
+  version = "v3.0.0";
```

Comment:
The version of the name [must start with a digit](http://nixos.org/nixpkgs/manual/#sec-package-naming), otherwise nix won't parse this correctly (for `nix-env -u` and `builtins.parseDrvName`) You should add the v prefix to the revision.